src/grader/grader_cli.ml

```diff
@@ -96,7 +96,7 @@ let grade exercise_dir output_json =
        let exercise_dir = remove_trailing_slash exercise_dir in
        read_exercise exercise_dir >>= fun exo ->
        let code_to_grade = match !grade_student with
-         | Some path -> read_student_file exercise_dir path
+         | Some path -> read_student_file (Sys.getcwd ()) path
```

Comment:
Is that a bugfix?

Reply:
yes, previously the argument would be checked as an existing file in CWD, then used as a path relative to `exercise_dir`. This makes the two consistent, with a more standard interface.